storage-proofs/porep/src/stacked/vanilla/create_label.rs

```diff
@@ -70,7 +70,7 @@ pub fn create_label_exp<H: Hasher>(
     // hash parents for all non 0 nodes
     let hash = if node > 0 {
         // prefetch previous node, which is always a parent
-        let prev = &layer_labels[(node - 1) * NODE_SIZE..node * NODE_SIZE];
+        let prev = &exp_parents_data[(node - 1) * NODE_SIZE..node * NODE_SIZE];
```

Comment:
This is not directly from the audit, but related to FPS-19.  Review is needed if the data should be explicitly pulled from the parents, which the comment indicates, or if we can grab it from the layer_labels.  They are both identical slices at this point, so it's a cosmetic update for now.

Reply:
^ Reverted/removed